app/assets/stylesheets/challenges.scss

```
@@ -0,0 +1,18 @@
+@import "bootstrap/scss/bootstrap";
+
+.challenges_title {
+  text-align: center;
+}
+
+.challenges_coment {
+  text-align: center;
+}
+
+.cards {

```

`.card` と混同しやすいのは避けた方がよいでしょう。

```suggestion
.text-cards { 
```